src/overlay/Tracker.h

```diff
@@ -39,6 +39,14 @@ using AskPeer = std::function<void(Peer::pointer, Hash)>;
 
 class Tracker
 {
+  public:
+    struct EnvelopeToTrack
+    {
+        Hash hash;
+        SCPEnvelope envelope;
+        Peer::MessageTrackerPtr cb;
```

Comment:
calling this `cb` is a little obscure. Maybe like `fetchProvokingMessage`?

(Assuming that's what it is, and not the tracker for the fetch message itself)

Reply:
I renamed this one to `SCPMsgTracker`: it seemed a bit more natural given that it tracks an SCP message from start to end (with or without fetching involved)